include/deal.II/fe/fe_series.h

```diff
@@ -115,6 +115,24 @@ namespace FESeries
               const unsigned int            cell_active_fe_index,
               Table<dim, CoefficientType> & fourier_coefficients);
 
+    /**
+     * Calculate all transformation matrices to transfer the finite element
+     * solution to the series expansion representation.
+     *
+     * These matrices will be generated on demand by calling calculate().
+     * However with this function, all matrices will be calculated in advance.
```

Comment:
Can you say what the point of the function is? If `calculate()` calls it anyway, why expose it to the public interface?